nuxeo-liveconnect-box-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-liveconnect/nuxeo-liveconnect-box-link.html

```diff
@@ -45,6 +45,17 @@
         @apply --layout-center;
         @apply --layout-center-justified;
         @apply --nuxeo-liveconnect-anchor-layout;
+        color: var(--nuxeo-secondary-color, #0066ff);
```

Comment:
This should be before the @apply instructions otherwise it won't be possible to override it (e.g. using `nuxeo-liveconnect-anchor-layout`)

Reply:
Also, we should include "nuxeo-styles" (```<style include="nuxeo-styles">```) when referring variables from [base theme](https://github.com/nuxeo/nuxeo-web-ui/blob/master/themes/base.html)